api/types/types.go

```diff
@@ -146,7 +146,7 @@ type Commit struct {
 // GET "/info"
 type Info struct {
 	ID                 string
-	Builder            BuilderVersion
+	Builder            string `json:",omitempty"`
```

Comment:
Was also wondering if we want to move this field further down if we keep this field